tests/PedantryTest.php

```diff
@@ -32,10 +32,8 @@ public function testMethodsAreOrderedAlphabeticallyByVisibility($className): voi
 
         $methods = array_filter(
             $methods,
-            function (ReflectionMethod $method) use ($class) {
-                return $method->getDeclaringClass() == $class // Exclude inherited methods
-                    && $method->getFileName() === $class->getFileName(); // Exclude methods inherited from traits
-            },
+            fn (ReflectionMethod $method) => $method->getDeclaringClass() == $class // Exclude inherited methods
+                    && $method->getFileName() === $class->getFileName(),
```

Comment:
You lost the 2nd comment
```suggestion
                    && $method->getFileName() === $class->getFileName(), // Exclude methods inherited from traits
```

Reply:
Lesson learned: don't trust `phpcbf` to preserve your comments.